tools/anthosbm-ansible-module/roles/kubectl-tool/tasks/main.yml

```diff
@@ -13,11 +13,6 @@
 # limitations under the License.
 
 ---
-- name: Remove kubectl if present
```

Comment:
The reason for removing the package before installation is to keep the role idempotent. Please keep this task.